stores/dynamodb/package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "@mastra/dynamodb",
+  "version": "0.0.1",
+  "description": "DynamoDB storage adapter for Mastra",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist",
+    "src"
+  ],
+  "scripts": {
+    "build": "tsup src/index.ts --format esm,cjs --clean --treeshake=smallest --splitting",
+    "dev": "tsup --watch",
+    "clean": "rm -rf dist",
+    "lint": "eslint .",
+    "test": "vitest run",
+    "test:watch": "vitest watch",
```

can we add some pre and post scripts to start the docker container?
```suggestion
    "pretest": "docker compose up -d",
    "test": "vitest run",
    "posttest": "docker compose down -v",
    "pretest:watch": "docker compose up -d",
    "test:watch": "vitest watch",
    "posttest:watch": "docker compose down -v",
```